packages/@icon-magic/generate/test/fixtures/out/home-2/ads/active-small.svg

```diff
@@ -1,4 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" class="flip" id="ads-active-small" aria-hidden="true" role="none" data-supported-dps="24x24" fill="currentColor">
   <path d="M12 17.13a5.13 5.13 0 010-10.25V4a8 8 0 108 8h-2.87A5.13 5.13 0 0112 17.13z" style="isolation:isolate" opacity=".75"/>
-  <path d="M23.31 6.34l-4.95-.71-.7-4.94-3.54 3.53.35 2.48-3.89 3.89a2 2 0 002.83 2.83l3.89-3.89 2.48.35z"/>
```

Comment:
Found out these paths come from this test:

```
    // Run generate again, hashing is true BUT the svg has been changed so plugins should run.
    const newSvg = `<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
      <title>24dp</title>
      <path d="M12,17.13A5.13,5.13,0,0,1,12,6.88V4a8,8,0,1,0,8,8H17.13A5.13,5.13,0,0,1,12,17.13Z" style="opacity: 0.75;isolation: isolate"/>
    </svg>`;
    const inputSvgPath = path.resolve(buildInput, `active-small.svg`);
    fs.writeFileSync(inputSvgPath, newSvg);
    ```
    
    So it is expected that the path gets removed!  Again I don't know how this path was not removed previously. I suspect hashing.